ddtrace/tracer/transport.go

```diff
@@ -93,12 +92,9 @@ func newHTTPTransport(addr string, client *http.Client) *httpTransport {
 		"Datadog-Meta-Tracer-Version":   version.Tag,
 		"Content-Type":                  "application/msgpack",
 	}
-	f, err := os.Open("/proc/self/cgroup")
-	if err == nil {
-		if id, ok := readContainerID(f); ok {
-			defaultHeaders["Datadog-Container-ID"] = id
-		}
-		f.Close()
+	cid := internal.ContainerID()
+	if cid != "" {
```

Comment:
We can merge this into one:
```suggestion
	if cid := internal.ContainerID(); cid != "" {
```